amr-wind/incflo_advance.cpp

```diff
@@ -564,7 +565,7 @@ void incflo::ApplyCorrector()
     // *************************************************************************************
     bool incremental = false;
     ApplyProjection(
-        (density_nph).vec_const_ptrs(), new_time, m_time.deltaT(), incremental);
+        (density_new).vec_const_ptrs(), new_time, m_time.deltaT(), incremental);
```

Comment:
Should that remain density_nph ?

Reply:
as we have discussed, density_new is intended, and it corresponds to the rest of the changes.